api/main.go

```diff
@@ -31,14 +18,6 @@ type ErrorResponse struct {
 
 // RootHandler handles get requests to the / endpoint.
 func RootHandler(ctx echo.Context) error {
-
-	// Extract the days query parameter.
-	days, err := util.IntQueryParam(ctx, "days", 1, 0, 365)
-	fmt.Println(days)
-	if err != nil {
-		return ctx.JSON(http.StatusBadRequest, ErrorResponse{Description: err.Error()})
-	}
-
```

Comment:
These changes (both in this file) look unrelated -- what's going on there?

Reply:
Those were for a testing parameter I was using to get it working on the root response. Just thought I should remove them now. And yes the println statements have been replaced by log statements